src/library_html5.js

```diff
@@ -152,6 +152,13 @@ var LibraryHTML5 = {
     },
     
     registerOrRemoveHandler: function(eventHandler) {
+      if (!eventHandler.target) {
+#if ASSERTIONS
+        console.error('registerOrRemoveHandler: the target element for event handler registration does not exist, when processing the following event handler registration:');
+        console.dir(eventHandler);
```

Comment:
Do we do this in any other places?  Wouldn't it be easy enough for the developer to see the object in devtools even without this?

Reply:
By the time the log happens, the object is long gone.